en/hardware/reference_design.md

```diff
@@ -17,3 +17,16 @@ FMU generations 1-3 were designed as open hardware, while FMU generations 4 and
 * FMUv4 PRO: Drotek Pixhawk 3 PRO \(STM32F469, 180 MHz, 384 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\)
 * FMUv5: Holybro Pixhawk 4 \(STM32F765, 216 MHz, 512 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1-n0__BYDedQrc_2NHqBenG1DNepAgnHpSGglke-QQwY/edit#gid=912976165)\)
 * FMUv6: work in progress, final name TBD, variant 6s \(STM32H7, 400 MHz, 2 MB RAM,  2 MB flash\) and variant 6i \(i.MX RT1050, 600 MHz, 512 KB RAM, external flash\)
+
+
+## Main/IO Function Breakdown
+
+The diagram below shows the division of bus and functional responsibilities between the FMU and I/O boards in a Pixhawk-series flight controller. 
+
+![PX4 Main/IO Function Breakdown main and io functions](../../assets/diagrams/px4_main_io_functions.png)
+
+Note that the I/0 board is optional; some Pixhawk-series controllers are built without the I/O board in order to save space.
```

Comment:
It's not only a question of space, but also:
- complexity
- cost
- use-case of the board